app/code/Magento/Catalog/Model/Attribute/ScopeOverriddenValue.php

```diff
@@ -120,6 +123,12 @@ public function getDefaultValues($entityType, $entity)
      */
     private function initAttributeValues($entityType, $entity, $storeId)
     {
+	if ($this->loadedEntityId !== $entity->getId()) {
+	    $this->attributesValues = [];
+	    $this->loadedEntityId = $entity->getId();
+	}
```

Comment:
The coding standard requires spaces as indentation. Please can you run `vendor/bin/phpcbf --standard=Magento2Framework app/code/Magento/Catalog/Model/Attribute/ScopeOverriddenValue.php` to fix this up.